src/Security/KeycloakAuthenticator.php

```diff
@@ -75,11 +78,16 @@ public function authenticate(Request $request): Passport
                 if ($user) {
                     $user->oauthKeycloakId = $keycloakUser->getId();
 
+                    $this->entityManager->persist($user);
```

Comment:
this was missing from keycloak but appears in all the other authenticators, so seemed like it might be needed